release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -280,6 +280,13 @@ In {product-title} {product-version}, you can use projects and categories to org
 
 For more information, see xref:../installing/installing_nutanix/installing-nutanix-installer-provisioned.adoc#installation-configuration-parameters-additional-vsphere_installing-nutanix-installer-provisioned[Installing a cluster on Nutanix].
 
+[id="ocp-4-13-installation-agent-console-application"]
+==== Agent-based Installer now performs network connectivity checks
+
+For installations of {product-title} {product-version} using the Agent-based Installer, an agent console application performs a pull check early in the installation process to verify that the current host can retrieve release images from the configured registry. The console application allows users to fix host network configuration issues before they might cause an installation to fail.
```

Comment:
```suggestion
For installations of {product-title} {product-version} using the Agent-based Installer, a console application (with a textual user interface) performs a pull check early in the installation process to verify that the current host can retrieve the configured release image. The console application supports the user in troubleshooting the problem in case of issue, and it also allows to modify directly the network configuration in case of need.
```

Reply:
Just my suggestions, feel free to rework them as per your convenience